src/not-so-smart/smart_git.ml

```diff
@@ -436,7 +436,7 @@ struct
       | None ->
           Log.debug (fun m -> m "End of pack.");
           emitter#close;
-          Lwt.return_unit
+          Lwt.pause ()
```

Comment:
this one we verified is not needed...

Reply:
Yes, I forget to remove it. I will delete it.